src/utils/helpers.js

```diff
@@ -77,6 +77,84 @@ var helpers = {
         return [hrz_altitude, hrz_azimuth]
     },
 
+    siderealTimeWithDate: function(lon, date) {
+        /* Local Sidereal Time with reference to J2000
+        *(edited to accept date)
+        *
+        * Equations courtesy of www.stargazing.net/kepler/altaz.html 
+        * and www.aberdeenastro.org.uk/sidereal_time.htm
+
+        *  LST = 100.46 + 0.985647 * d + lon + 15*UT
+        *
+        *       d    is the days from J2000, including the fraction of a day
+        *       UT   is the universal time in decimal hours
+        *       lon is your longitude in decimal degrees, East positive.
+        *       date is a Date object, e.g. "2021-07-28T07:00:00.000Z"
```

Comment:
Maybe say this is a javascript Date object, e.g. `new Date()`. Having an example is good but in this case the example looks like a string rather than a date object.